features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandler.kt

```diff
@@ -34,15 +36,18 @@ internal class RecordedDataQueueHandler {
     private var processor: RecordedDataProcessor
     private var rumContextDataHandler: RumContextDataHandler
     private var timeProvider: TimeProvider
+    private var base64LruCache: Cache
 
     internal constructor(
         processor: RecordedDataProcessor,
         rumContextDataHandler: RumContextDataHandler,
-        timeProvider: TimeProvider
+        timeProvider: TimeProvider,
+        base64LruCache: Cache = Base64LRUCache
```

Comment:
minor: `Cache` type doesn't say that it is `LRU`, so to not leak the exact implementation and avoid confusion in the future (maybe at some point we switch from `LRU` do something else), I propose to rename `base64LruCache` to just `base64Cache` everywhere in this class.